plugins/GSdx/GSRendererHW.h

```diff
@@ -96,31 +96,31 @@ class GSRendererHW : public GSRenderer
 
 		template<class T> class FunctionMap : public GSFunctionMap<uint32, T>
 		{
-			list<HackEntry<T> >& m_tbl;
+			std::list<HackEntry<T> >& m_tbl;
 
 			T GetDefaultFunction(uint32 key)
 			{
 				CRC::Title title = (CRC::Title)(key & 0xffffff);
 				CRC::Region region = (CRC::Region)(key >> 24);
 
-				for(typename list<HackEntry<T> >::iterator i = m_tbl.begin(); i != m_tbl.end(); i++)
+				for(const auto &i : m_tbl)
```

Comment:
I know it's being pedant, but maybe renaming `i` to something more related to the list element type is better, like `entry`, to remark that the variable is not an `iterator` but a `HackEntry<T>&`. Here maybe it's useless, but this applies in general where the iterator has been substitued by a entry reference or entry copy in the new foreach loops.

Reply:
I made the change in this case (and one other I think). The rest I left alone since I don't think it would improve readability (think they mostly/all involve map/unordered_map).